sys/event/event.c

```diff
@@ -106,30 +106,57 @@ static event_t *_wait_timeout(event_queue_t *queue, xtimer_t *timer)
     do {
         result = event_get(queue);
         if (result == NULL) {
-            flags = thread_flags_wait_any(THREAD_FLAG_EVENT | THREAD_FLAG_TIMEOUT);
+            flags = thread_flags_wait_any(THREAD_FLAG_EVENT |
+                                          THREAD_FLAG_TIMEOUT);
         }
     } while ((result == NULL) && (flags & THREAD_FLAG_EVENT));
 
+    return result;
+}
+#endif
+
+#if IS_USED(MODULE_XTIMER)
+static event_t *_wait_timeout_xtimer(event_queue_t *queue, xtimer_t *timer)
+{
+    event_t *result = _wait_timeout(queue);
     if (result) {
         xtimer_remove(timer);
     }
-
```

Comment:
Unrelated, is this related to uncrustifying the file? If so can you split into its own commit?